src/engine/structures.rs

```diff
@@ -43,6 +43,26 @@ where
     }
 }
 
+impl<U, T> Extend<(Name, U, T)> for Table<U, T>
```

Comment:
The code is good, but I'm convinced it's dead; should remove or add tests.

Reply:
Whoops! It looks like the code I was using that for got removed.